### DISCUSSION TITLE
[Adapter implementation] How to remove output from successful test?

### DISCUSSION THREAD

**Discussion by sidlatau** [nvim-neotest/neotest#45] [Q&A] [answered]
I can not figure out how to remove ability to display output if test is successful. Now if I do not provide `neotest.Result.output` property, it displays all test file output:
<img width="1095" alt="image" src="https://user-images.githubusercontent.com/4973296/176280720-203bf843-d61b-4594-b983-f127d495ffa8.png">
I may provide empty file, but then it looks strange:
<img width="1023" alt="image" src="https://user-images.githubusercontent.com/4973296/176281266-aca0adfb-4681-4121-903e-76e16ed0354e.png">
Is it possible to remove ability to open popup if there is no output for the test?

**Comment by rcarriga** [comment, marked as answer]
It is not possible because the user may still want to see the output of the command so neotest defaults to the output of the process. Why not let the user see the output? If it's because of the JSON, you can construct the output like neotest-go does https://github.com/nvim-neotest/neotest/issues/6#issuecomment-1150341538.

**Reply by sidlatau** [reply]
Thanks for the hint to use tempfile.
